### ISSUE TITLE
npm install fails again

### ISSUE THREAD

**Issue by chenchaoyi** [ErikDubbelboer/node-sleep#36]
Seems a nan breaking change again:

```
$ npm install sleep
|
> sleep@3.0.0 install /private/tmp/node_modules/sleep
> node-gyp rebuild

  CXX(target) Release/obj.target/node_sleep/sleep.o
In file included from ../sleep.cc:2:
In file included from ../node_modules/nan/nan.h:2104:
../node_modules/nan/nan_weak.h:363:42: error: cannot create a non-constant pointer to member function
    v8::PersistentBase<T>::SetWeak(wcbd, wcbd->invoke);
                                         ^~~~~~~~~~~~
../node_modules/nan/nan_object_wrap.h:56:18: note: in instantiation of function template specialization
      'Nan::Persistent<v8::Object, v8::NonCopyablePersistentTraits<v8::Object> >::SetWeak<Nan::ObjectWrap>' requested here
    persistent().SetWeak(this, WeakCallback, WeakCallbackType::kParameter);
                 ^
In file included from ../sleep.cc:2:
In file included from ../node_modules/nan/nan.h:2104:
../node_modules/nan/nan_weak.h:378:42: error: cannot create a non-constant pointer to member function
    v8::PersistentBase<T>::SetWeak(wcbd, wcbd->invoke);
                                         ^~~~~~~~~~~~
2 errors generated.
make: *** [Release/obj.target/node_sleep/sleep.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Darwin 13.4.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /private/tmp/node_modules/sleep
gyp ERR! node -v v0.12.7
gyp ERR! node-gyp -v v2.0.1
gyp ERR! not ok
npm ERR! Darwin 13.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "sleep"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3
npm ERR! code ELIFECYCLE

npm ERR! sleep@3.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sleep@3.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the sleep package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls sleep
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /private/tmp/npm-debug.log
```

```
$ npm version
{ npm: '2.11.3',
  http_parser: '2.3',
  modules: '14',
  node: '0.12.7',
  openssl: '1.0.1p',
  uv: '1.6.1',
  v8: '3.28.71.19',
  zlib: '1.2.8' }
```

Mac OS 10.9.5

**Comment by jhigueras** [comment]
Same here :/

{ npm: '2.7.4',
  http_parser: '2.3',
  modules: '14',
  node: '0.12.2',
  openssl: '1.0.1m',
  uv: '1.4.2-node1',
  v8: '3.28.73',
  zlib: '1.2.8' }

**Comment by chenchaoyi** [comment]
This is fixed as as nan@2.0.4 is released....

**Comment by jhigueras** [comment]
Perfect, now it works :D
